src/Microsoft.Health.Fhir.CosmosDb.UnitTests/Features/Storage/CosmosFhirDataStoreTests.cs

```diff
@@ -183,6 +199,26 @@ public async Task GivenAQuery_WithPagesWithFewResults_GivesUpAfterHalfTheResults
             Assert.Equal("5", continuationToken);
         }
 
+        [Fact]
```

Comment:
Should there be a test for the request timeout exception that was added?

Reply:
Good call @LTA-Thinking !  Added the test case and found a bug.